tests/backends/oracle/tests.py

```diff
@@ -69,6 +70,63 @@ def test_check_database_version_supported(self, mocked_get_database_version):
             connection.check_database_version_supported()
         self.assertTrue(mocked_get_database_version.called)
 
+    @unittest.skipUnless(connection.is_pool, "Pool specific tests")
+    def test_connect_pool_set_to_true(self):
+        test_pool = connection.settings_dict["OPTIONS"]["pool"]
+        self.assertTrue(test_pool)
+
+    @unittest.skipUnless(connection.is_pool, "Pool specific tests")
+    def test_connect_pool(self):
```

Comment:
This test hangs.

Reply:
The test was hanging because the default value for min and max connections in oracledb is 1. As the test was trying to hold two connections but max was not specified in the options dictionary of the settings.py file, it caused the program to hang. To resolve this issue, I modified the code to set max as 4 by default if it is not specified in the options dictionary.